requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy~=1.20.2
+h5py~=3.2.1
```

Comment:
How did you pick these version numbers?

Reply:
They were generated by my IDE, they correspond to the versions I was using when making changes. I just replaced the "~=" by ">=" and filled the missing versions number.

Reply:
Well, if they are not the minimal version supported, I would rather have no version number. I will remove them.